dotCMS/src/main/java/com/dotcms/contenttype/business/ContentTypeAPIImpl.java

```diff
@@ -772,7 +784,30 @@ public List<ContentType> search(final List<String> sites, final String condition
     }
   }
 
-    @CloseDBIfOpened
+  /**
+   * Utility method that takes a list of Site IDs and/or Site Keys, and resolves them to a list of
+   * Site IDs only. This allows developers/users to pass down a Site Key when they don't have
+   * access to its ID, or just don't want to use it.
+   *
+   * @param sites The list of Site IDs and/or Site Keys to resolve.
+   *
+   * @return A list of Site IDs only.
+   */
+  private List<String> resolveSiteIds(final List<String> sites) {
```

Comment:
nice

Reply:
I think this method could be promoted to some SiteUtils or so

Reply:
Good idea! I'll make this specific change in the other PR I have

Reply:
Done!